app/http/middleware/VerifyEmailMiddleware.py

```diff
@@ -0,0 +1,17 @@
+''' Verify Email Middleware '''
+from masonite.request import Request
+
+class VerifyEmailMiddleware:
+    ''' Verify Email Middleware '''
+
+    def __init__(self, request: Request):
+        self.request = request
```

Comment:
Let's just clean this class up with the new docstrings we are using with other classes that look something like:

```
"""Short description here

Arguments:
    {something} here

...
```